src/qt/createassetdialog.cpp

```diff
@@ -451,19 +451,19 @@ bool CreateAssetDialog::checkIPFSHash(QString hash)
 {
     if (!hash.isEmpty()) {
         std::string error;
-        if (!CheckEncodedIPFS(hash.toStdString(), error)) {
+        if (!CheckEncoded(DecodeAssetData(hash.toStdString()), error)) {
             ui->ipfsText->setStyleSheet("border: 2px solid red");
-            showMessage("IPFS Hash must start with 'Qm'");
+            showMessage("IPFS Hash must start with 'Qm' or txid's must have size of 64 characters");
             disableCreateButton();
             return false;
         }
-        else if (hash.size() != 46) {
+        else if (hash.size() != 46 && hash.size() != 64) {
             ui->ipfsText->setStyleSheet("border: 2px solid red");
-            showMessage("IPFS Hash must have size of 46 characters");
+            showMessage("IPFS/txid Hash must have size of 46 characters");
```

Comment:
`IPFS/Txid` to remain consistent with other usages thus far

Reply:
Fixed